.circleci/config.yml

```diff
@@ -92,11 +99,13 @@ workflows:
           requires:
             - dependencies
             - lint
+      - test_examples
```

Comment:
We should wait until the `dependencies`, `lint` and `test` jobs have finished successfully before running the examples.

We want to "fail fast" on CI so we're not blocking other builds unnecessarily.

This should look something like:

```yaml
- test_examples
    requires:
      - test
```

Reply:
Good call!